package.json

```diff
@@ -47,4 +48,4 @@
     "build/**",
     "contract-bindings/**"
   ]
-}
+}
```

Comment:
Could you keep the newline to avoid unnecessary diff noise?

Reply:
That was unintentional, sorry. Will check my editor :eyes:

Fixed btw